test/functional/readconcern.test.js

```diff
@@ -59,9 +59,12 @@ describe('ReadConcern', function() {
   ];
 
   tests.forEach(test => {
+    const metadata = { requires: { topology: 'replicaset', mongodb: '>= 3.2' } };
+    if (test.commandName === 'geoSearch') {
+      metadata.requires.mongodb += ' <=4.4';
+    }
```

Comment:
This is a data-driven test, shouldn't the instructions for building the tests should be in the specs above rather than special cased here? Maybe something like `serverVersion: '<=4.4'`?